DeUrgenta/Features/AccountConfig/View/AccountConfigView.swift

```diff
@@ -1,7 +1,12 @@
 import SwiftUI
 
+protocol AccountConfigViewDelegate {
+    func accountConfigViewShouldRedirect(from: AccountConfigView, withItem: ConfigStep)
```

Comment:
`func accountConfigViewShouldRedirect(from view: AccountConfigView, withItem item: ConfigStep)`
Swift has argument labels - https://www.swift.org/documentation/api-design-guidelines/#argument-labels - this will make it more clear.

Reply:
I understand